sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+* **Bootstrapping network growth flywheel (including grants, bounties, user incentives, business development for key integrations & partnerships, and M\&A): 31%** Distinct from marketing, bootstrapping budget covers a key category of funds that will be applied to user growth efforts both at the Stacks and application levels. As Stacks builders look to grow their applications, bootstrapping the necessary protocol/ecosystem level infrastructure is initially very expensive. This budget ensures the fundamental pieces that builders rely on ecosystems in our industry are provided, and Stacks can be even more competitive for builder talent. Industry expectations are that many of these incentives are provided by ‘the ecosystem’, meaning that investors all expect this when considering investing in Stacks startups. The primary focus of these funds is to support builders as they reach the point where they require growth incentives, and this should not only make it faster for apps to reach critical mass, but also to secure investors on the way. Importantly, this category also includes funds to support aggressive and expedited business development momentum for the Stacks chain, potential M\&As, and other activities under the ‘Business Development’ umbrella, further enabling builders with the interoperability and support their apps need to grow. Examples of spend may include integration costs for bridges, stablecoins, sBTC listings, more fiat on/off ramps, oracle and node providers, incentives for Signers, and more.  
+* **Working Capital: 27%** Working capital describes capital that is intended to be actively deployed, but not permanently spent. This includes capital liquidity provisioning, a Stacks Memecoin Fund, direct deployments into Stacks DeFi, and allocations to deals that require a certain capital allocation to incentivize integration into the Stacks ecosystem.
+
+### **Benefits To Builders**
+
+With the majority of the Endowment dedicated to empowering builders of all sizes and fueling growth, the Stacks ecosystem is entering its next wave of innovation. To provide more specific examples, Stackers can expect to see activities like the following supported by this new endowment (in no particular order):
```

Comment:
> the Stacks ecosystem is entering its next wave of innovation

This is unquantifiable and unfalsifiable marketing-speak.  Suggest dropping.

Reply:
I think the whole SIP-031 implies it is taking the Stacks to the (wave of innovation) next level already.

And this section is really just about "_benefits to builders_", I think it would read a lot clearer for the builders if follows Jude's suggestion to remove - so I agree. 

Would read like below:

`With the majority of the Endowment dedicated to empowering builders of all sizes and fueling growth. To provide more specific examples, Stackers can expect to see activities like the following supported by this new endowment (in no particular order):`